### DISCUSSION TITLE
What is the keyboard shortcut to open up search in the generated HTML?

### DISCUSSION THREAD

**Discussion by bkowshik** [quarto-dev/quarto-cli#5808] [Q&A] [answered]
### Description

Is there a keyboard shortcut to open up `Search` instead of clicking on the search icon in the quarto generated HTML files? This would help avoid clicking and instead have a quick keyboard shortcut key to search through the HTML pages.

**Comment by dragonstyle** [comment, marked as answer]
Recent builds of Quarto 1.4 include default search shortcuts of `f` and `/`. You can customize this using `search` > `keyboard-shortcut` to override with your own key(s). 


https://github.com/quarto-dev/quarto-cli/commit/d90039f2f6f24385334bb3ceb9b690f9884a72fc
